src/components/global/ContactFormModal.jsx

```diff
@@ -15,27 +16,32 @@ import emailjs from '@emailjs/browser';
 import { PropTypes } from 'prop-types';
 
 const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const [messageSuccess, setMessageSuccess] = useState(false);
+  const [messageFailure, setMessageFailure] = useState(false);
   const form = useRef();
 
   const handleClose = () => {
+    setMessageSuccess(false);
+    setMessageFailure(false);
     setShowContactFormModal(false);
   };
 
   const handleSubmit = (e) => {
     e.preventDefault();
 
     emailjs.sendForm('service_co2agxf', 'template_20pnwni', form.current, 'AkBl59Ya3226OfPyQ').then(
```

Comment:
Initially, I was going to say we should protect these keys, but after reading the docs, I don't think it's really an issue. Someone could grab the keys but can only send our pre-defined templates so there is no value in it for them. 

https://www.emailjs.com/docs/faq/is-it-okay-to-expose-my-public-key/

Reply:
Yeah exposing the keys isn't an issue. Besides, we'll be creating a completely new account just for CODE PDX. This one is just for testing.

And I tested moving the keys into a `.env` file. It's definitely doable but takes a bit more effort than I felt was inside the scope of this PR. We can create a new one to do that if we want. It'd be best practice, but in this case doesn't compromise security.